pull_request_template

```diff
@@ -0,0 +1,42 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
```

Comment:
I'd like to suggest we use the formal "draft pull request" to indicate that a pull request is not ready; and other wise the popular convention of "WIP" in the title.